frontend/src/store/alert-store.js

```diff
@@ -4,13 +4,22 @@ const STORE_VARIABLES = {
   AlertDetails: {
```

Comment:
@jagadeeswaran-zipstack Can we change `AlertDetails` to camel case here and all its corresponding places?

Reply:
sure